app/src/lib/dispatcher/validated-repository-path.ts

```diff
@@ -1,15 +1,12 @@
-import * as Path from 'path'
-
-import { getGitDir } from '../git'
+import { getTopLevelWorkingDirectory } from '../git'
 
 /**
  * Get the path to the parent of the .git directory or null if the path isn't a
  * valid repository.
  */
 export async function validatedRepositoryPath(path: string): Promise<string | null> {
   try {
-    const gitDir = await getGitDir(path)
```

Comment:
I think we should be able to get rid of `getGitDir`. I doubt we're using it anywhere else.